### PR TITLE
Dashboard support on Infrequent tier

### PR DESCRIPTION
**Pull request by hganapathy**
## Purpose of this pull request

- Documentation for dashboard support on Infrequent Tier on Limited Availability
- This page should not be searchable in the help docs

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/beta/infrequent-dashboard-support.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+<p> <a href="/docs/beta"><span className="beta">Beta</span></a> </p>
+
+Previously, dashboards were only supported in Sumo Logic’s Continuous data tier. Now, you can also setup dashboard panels that run against the Infrequent tier.
```
Comment (JainM6) [inline docs/beta/infrequent-dashboard-support.md:15]: I'd rephrase it as:
Previously, dashboards were only supported in Sumo Logic's Continuous data tier. Now, you can also set up dashboard panels that run against the Infrequent tier, making it easier to include data from the Infrequent tier in the scope of dashboard panels.
```

File: docs/beta/infrequent-dashboard-support.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+## Sample Queries
+
+This means you can now include data from the Infrequent Tier in the scope of dashboard panels.
```
Comment (JainM6) [inline docs/beta/infrequent-dashboard-support.md:19]:
```suggestion
To use data from the Infrequent tier in your dashboard panels, you can use the following sample queries
```

File: docs/beta/infrequent-dashboard-support.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+For example, you can use `_dataTier=Infrequent` in your query scope:
+
+```sql
+_dataTier=Infrequent _sourceCategory=appA "error"
+```
+
```
Comment (JainM6) [inline docs/beta/infrequent-dashboard-support.md:26]:
```suggestion
```

File: docs/beta/infrequent-dashboard-support.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+<p> <a href="/docs/beta"><span className="beta">Beta</span></a> </p>
+
+Previously, dashboards were only supported in Sumo Logic’s Continuous data tier. Now, you can also setup dashboard panels that run against the Infrequent tier.
+
+## Sample Queries
+
+This means you can now include data from the Infrequent Tier in the scope of dashboard panels.
+
+For example, you can use `_dataTier=Infrequent` in your query scope:
```
Comment (JainM6) [inline docs/beta/infrequent-dashboard-support.md:21]:
```suggestion
 * Use `_dataTier=Infrequent` in your query scope. 
```

File: docs/beta/infrequent-dashboard-support.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+</head>
+
+<p> <a href="/docs/beta"><span className="beta">Beta</span></a> </p>
+
+Previously, dashboards were only supported in Sumo Logic’s Continuous data tier. Now, you can also setup dashboard panels that run against the Infrequent tier.
+
+## Sample Queries
+
+This means you can now include data from the Infrequent Tier in the scope of dashboard panels.
+
```
Comment (JainM6) [inline docs/beta/infrequent-dashboard-support.md:20]:
```suggestion
```

File: docs/beta/infrequent-dashboard-support.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+For example, you can use `_dataTier=Infrequent` in your query scope:
+
```
Comment (JainM6) [inline docs/beta/infrequent-dashboard-support.md:22]:
```suggestion
```

File: docs/beta/infrequent-dashboard-support.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+```sql
+_dataTier=Infrequent _sourceCategory=appA "error"
+```
+
+Or, you can select data in the Infrequent tier by specifying the Partition that contains it, like this:
```
Comment (JainM6) [inline docs/beta/infrequent-dashboard-support.md:27]:
```suggestion
Or select data in the Infrequent tier by specifying the Partition that contains it. For example:
```

File: docs/beta/infrequent-dashboard-support.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+
```
Comment (JainM6) [inline docs/beta/infrequent-dashboard-support.md:28]:
```suggestion
```

File: docs/beta/infrequent-dashboard-support.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+```sql
+_index=some_infrequent_index
+```
+
```
Comment (JainM6) [inline docs/beta/infrequent-dashboard-support.md:32]:
```suggestion
```

File: docs/beta/infrequent-dashboard-support.md [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+or
+
```
Comment (JainM6) [inline docs/beta/infrequent-dashboard-support.md:34]:
```suggestion
```

File: docs/beta/infrequent-dashboard-support.md [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+_dataTier=Infrequent _sourceCategory=appA "error"
+```
+
+Or, you can select data in the Infrequent tier by specifying the Partition that contains it, like this:
+
+```sql
+_index=some_infrequent_index
+```
+
+or
```
Comment (JainM6) [inline docs/beta/infrequent-dashboard-support.md:33]:
```suggestion
Or
```

File: docs/beta/infrequent-dashboard-support.md [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+
+
+```sql
+_index=some_infrequent_index OR _index=some_continuous_index
+```
+
```
Comment (JainM6) [inline docs/beta/infrequent-dashboard-support.md:39]:
```suggestion
```

File: docs/beta/infrequent-dashboard-support.md [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+
+## Infrequent Dashboard Usage
+
+Sumo Logic provides a dashboard you can use to monitor the Infrequent tier consumption by saved Dashboards.
+
+<img src={useBaseUrl('img/beta/infrequent-dashboard.png')} alt="dashboard" />
+
+The dashboard presents the following information for the currently selected time range:
+
```
Comment (JainM6) [inline docs/beta/infrequent-dashboard-support.md:53]:
```suggestion
```

File: docs/beta/infrequent-dashboard-support.md [hunk 14]
```diff
@@ -0,0 +1,58 @@
...
+* **Avg Credits Consumed/Day**. The daily average number of credits consumed over the last 30 days.
+* **Total Credits Consumed**. The total credits consumed in the last 30 days.
+* **Top 50 Expensive Queries**. The top 50 expensive queries setup on Dashboards to run against Infrequent tier.
```
Comment (JainM6) [inline docs/beta/infrequent-dashboard-support.md:56]:
```suggestion
* **Top 50 Expensive Queries**. The top 50 expensive queries set up on Dashboards to run against the Infrequent tier.
```

**Review by JainM6 — CHANGES_REQUESTED** [review]
Minor formatting changes..

File: docs/beta/infrequent-dashboard-support.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (hganapathy) [reply docs/beta/infrequent-dashboard-support.md:15]: This seems to reiterate the same thing twice when I read it.

> ".. set up dashboard panels that run against the Infrequent tier"
> and "making it easier to include data from the Infrequent tier in the scope of dashboard panels"

Both of these to me appear like the same thing. Can we simplify the whole sentence instead to:

> "Previously, dashboards were only supported in Sumo Logic's Continuous data tier. This support enables you to also > set up dashboard panels to include data from the Infrequent tier."

[the remark this answers is not in the document]
Comment (JainM6) [reply docs/beta/infrequent-dashboard-support.md:15]: Yes, we can. This looks good.

**Review by JainM6 — APPROVED** [review]
